### ISSUE TITLE
Ghost markers when zooming out a chart with accessibility & styled mode & disabled markers

### ISSUE THREAD

**Issue by ddragula** [highcharts/highcharts#23878]
#### Expected behaviour
When zooming out the chart, no markers are visible, especially when `marker.enabled: false`.

#### Actual behaviour
When zooming out the chart with accessibility module on & styled mode on, disabled (ghost) markers are shown after zooming out.

#### Live demo with steps to reproduce
https://jsfiddle.net/BlackLabel/6a3o0jef/
1. zoom in
2. zoom out

See the ghost markers:
<img width="1112" height="385" alt="Image" src="https://github.com/user-attachments/assets/2351a499-862f-476d-8594-5cb05087987e" />

#### Workaround
If you don't want to disable styled mode or accessibility, set the `enabledThreshold` to a very large number so that they are never shown: https://jsfiddle.net/BlackLabel/Lvnbmrkf/

#### Product version
Highcharts v12.4
